layouts/partials/home/use-cases.html

```diff
@@ -0,0 +1,36 @@
+<section class="section has-text-centered use-cases">
+  <div class="container">
+
+    <div class="columns is-variable is-6">
+      <div class="column content is-medium">
+        <p class="image">
+            <a href="#">
+              <img src="/img/usecases/uc1-secure.svg">
+            </a>
+        </p>
+        <p>Secure microservices automatically with Envoy, X.509 PKI or JWT</p>
+      </div> <!-- .column -->
+
+      <div class="column content is-medium">
+        <p class="image">
+            <a href="#">
+              <img src="/img/usecases/uc2-svcs.svg">
+            </a>
+        </p>
+        <p>Authenticate securely to common databases or platforms, without passwords or API keys</p>
+      </div>
+
+      <div class="column content is-medium">
+        <p class="image">
+            <a href="#">
+              <img src="/img/usecases/uc3-mesh.svg">
+            </a>
+        </p>
+        <p>Build, bridge and extend service mesh across organizations without sharing keys</p>
```

Comment:
```suggestion
        <p>Build, bridge and extend service mesh across technology boundaries without sharing keys</p>
```

Reply:
The goal of this copy was to indirectly highlight the utility of federation as a technology that can allow trusted interop between different groups that may wish to securely authenticate (kinda to Evan's point above) without trusting a common key. Your proposal changes the meaning of this to focus on deployment in heterogenous environments instead. I'd like to see if there's a better way of doing that elsewhere in the homepage.

Reply:
Thanks for the explanation. I now see where you are going with it. 

The point is valid in the case of both organizations in the sense of administrative boundaries as you describe it as well as technological boundaries, or boundaries alone, whatever those may be.